src/Service.Tests/dab-config.MsSql.json

```diff
@@ -1209,6 +1209,14 @@
         "enabled": true
       },
       "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            {
+              "action": "read"
+            }
+          ]
+        },
```

Comment:
Is this required at this point in time since all tests are passing/ are not impacted - meaning they don't really have a dependency on this configuration?